samples/trumpify/trump.py

```diff
@@ -0,0 +1,83 @@
+import os
+import sys
+import time
+from threading import Thread
+
+from twitter import *
+
+sys.path.append(os.path.abspath('../../'))
+
+from daemo.client import Client
+
+CREDENTIALS_FILE = 'credentials.json'
+
+PROJECT_ID = -1
+
+TW_CONSUMER_KEY = ''
+TW_CONSUMER_SECRET = ''
+TW_ACCESS_TOKEN = ''
+TW_ACCESS_TOKEN_SECRET = ''
```

Comment:
The thing about oauth2 refresh tokens is that they change, and that wouldn't quite work with env variables because we would have to update those variables, we would need some sort of storage